orderbook/src/fee.rs

```diff
@@ -213,10 +221,35 @@ impl MinFeeCalculator {
                     })
                     .await?
                     .gas
-                    .to_f64_lossy()
+                    .to_f64_lossy();
+
+                match kind {
```

Comment:
nit: rather than logging here (I think this information could be conveyed together in the unconditional log below), I think it would be nice if we could log where the estimate comes from (e.g. if from Fallback, Baseline, Paraswap or DB)

Reply:
> we could log where the estimate comes from (e.g. if from Fallback, Baseline, Paraswap or DB)

So, in order to distinguish between the Baseline and ParaSwap solver, that would require changes to the priority price estimator. For now, you will see warnings with:
```
inner price estimator failed
```

If the ParaSwap price estimator failed during the fee estimate, meaning that the Baseline price estimator was used instead.

This is not ideal, but I will implement disambiguating between the different price estimate sources in a separate PR as it would touch code unrelated to this change (https://github.com/gnosis/gp-v2-services/issues/1265).